### DISCUSSION TITLE
Delayed OS updates for forward nodes

### DISCUSSION THREAD

**Discussion by Ximalas** [Security-Onion-Solutions/securityonion#13580] [Unsupported Versions] [answered]
I have only one forward node, but I keep noticing it sometimes lags behind the rest of the nodes as far as OS updates are concerned. See below for an example. I usually take notice in the morning and all nodes are usually ready to be rebooted by then. Is there a way I can accelerate the pre-update stage on the forward node?

![image](https://github.com/user-attachments/assets/5f1a3765-083a-4a24-8973-0ccc6cbfadee)

**Comment by reyesj2** [comment]
Each grid node automatically checks in every 8 hours with the manager for os updates. Is this sensor taking longer than ~24 hours to update?

**Comment by Ximalas** [comment]
All nodes are in the same VLAN within microseconds of reach to one another. I have nothing node specific configured when it comes to OS updates.

![image](https://github.com/user-attachments/assets/96834cf4-3329-4732-8295-6594e60cb94e)

**Reply by Ximalas** [reply, marked as answer]
As a curiosity, today I have the inverse situation.

![image](https://github.com/user-attachments/assets/a3af80fe-d4d5-4344-b7b2-85f606dd285e)

Maybe it's perfectly normal for some nodes to be earlier than others.

![image](https://github.com/user-attachments/assets/c5b22cd9-3057-4f2d-b25f-e273c8f5be40)
